.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  build_zips:
+    strategy:
+      fail-fast: true
```

Comment:
What's the purpose of `fail-fast`? Don't we want to build as many zips as possible?

Reply:
I thought that we need to re run the workflow when it failure. I’m not sure that the upload artifact can keep the success artifact. If it can keep, the fail fast may not need.

Reply:
Good point. Let's keep it.